src/hooks/use-heatmap/use-heatmap.js

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react';
+import { getHeatmapFromPosts } from './heatmap-fns';
+
+const useHeatmap = (posts) => {
+  const [heatmap] = useState(getHeatmapFromPosts(posts));
```

Comment:
Why do you store the mapped data in a state variable?

Reply:
This is most certainly an over-abstraction. I started with a hook without there being much justification for it in the end. I will look to simplify 😄